### ISSUE TITLE
Firefox not respecting width/height on SVG elements

### ISSUE THREAD

**Issue by indygriffiths** [silverstripe/silverstripe-mfa#333]
Browser: Firefox 68.0.2

I mean it's great that Firefox is enthusiastic about us setting up MFA but this may be a bit much.

![image](https://user-images.githubusercontent.com/32345766/63403986-985c3000-c435-11e9-8409-f217a8a68aad.png)

**Comment by indygriffiths** [comment]
Issue is that FIrefox cannot parse REM units in the attributes, but can if you specify them as CSS attributes. Looks like it's been fixed in the unreleased Firefox 69: https://bugzilla.mozilla.org/show_bug.cgi?id=1231147 and https://bugzilla.mozilla.org/show_bug.cgi?id=1383650

Works:
```
<svg height="96" width="96">
<svg height="96px" width="96px">
<svg height="2em" width="2em">
<svg height="8rem" width="8rem" style="width:8rem;height:8rem">
```

Fails:
```
<svg height=8rem" width="8rem">
```

**Comment by robbieaverill** [comment]
OK so we'll add some CSS rules for the SVGs we have in this module - hopefully we can fix #301 at the same time

**Comment by Cheddam** [comment]
Seems to only affect SS3, as we use admin's icon font in the SS4 variant.
